lightning/src/util/errors.rs

```diff
@@ -101,50 +101,3 @@ impl_writeable_tlv_based_enum_upgradable!(APIError,
 	(8, MonitorUpdateInProgress) => {},
 	(10, IncompatibleShutdownScript) => { (0, script, required), },
 );
-
-#[inline]
-pub(crate) fn get_onion_debug_field(error_code: u16) -> (&'static str, usize) {
-	match error_code & 0xff {
-		4 | 5 | 6 => ("sha256_of_onion", 32),
-		11 | 12 => ("htlc_msat", 8),
-		13 | 18 => ("cltv_expiry", 4),
-		19 => ("incoming_htlc_msat", 8),
-		20 => ("flags", 2),
-		_ => ("", 0),
-	}
-}
-
-#[inline]
-pub(crate) fn get_onion_error_description(error_code: u16) -> (&'static str, &'static str) {
```

Comment:
Remove in commit where display is added?

Reply:
Would need to remove in `process_onion_failure_inner` as well then, would rather touch that function all in one commit.